instrumentation/nethttp/status_code.go

```diff
@@ -7,10 +7,14 @@ import (
 
 type statusCodeTracker struct {
```

Comment:
Rename it to something else? Like `responseTracker`?

Reply:
done